src/components/video/Video.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+import {
+  StyledFigure,
+  VideoContainer,
+  PlaceholderButton,
+  StyledVideoImage,
+  StyledFigCaption,
+} from './Video.styles'
+import { VideoProps } from './Video.types'
+
+const Video = ({
+  videoId,
+  coverImage: { image, alt },
+  caption,
+}: VideoProps) => {
+  const [clicked, setClicked] = useState(false)
+
+  return (
+    <StyledFigure>
+      {clicked ? (
+        <VideoContainer>
+          <iframe
+            allowFullScreen
+            frameBorder="0"
+            src={`https://www.youtube.com/embed/${videoId}?autoplay=1`}
+            title={caption}
+          />
+        </VideoContainer>
+      ) : (
+        <>
+          <div onClick={() => setClicked(!clicked)}>
+            <StyledVideoImage fixed={image} alt={alt} />
+          </div>
```

Comment:
This can just be the `image` rather than a clickable `div` and an image. Should probably cahnge the `role` to be `button` as this is semantically correct. Note that doing this will disrupt your tests, so perhaps adding a title and using `findByTitle` could be used for the `img` in test?